.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Description of changes
+
+<!-- What have you implemented in this PR? -->
+
+## Checklist
+
+<!-- [x] if you have completed the task -->
+<!-- [n/a] if the task is not releveant -->
```

Comment:
- [ ] 100% coverage of new code 
- [ ] Pass Lint 
- [ ] Smoke tested locally using latest main from relevant repos  
- [ ] Package version updated